### PR TITLE
Fix common colorization issue in class/interface declaration.

### PR DESCRIPTION
**Pull request by hoanhtien**
Before:
image

After:
image

### CODE REVIEW

File: TypeScript.YAML-tmLanguage [hunk 1]
```diff
@@ -76,10 +76,21 @@ repository:
...
+    - include: '#object-heritage-parent'
+    - include: '#comment'
+  
+  object-heritage-parent:
+    name: meta.object.heritage.parent.ts
+    match: '(?:\s*([a-zA-Z_$][\w$]*))'
     captures:
-       '1': { name: keyword.other.ts }
-       '2': { name: storage.type.ts } 
+      '1': { name: storage.type.ts }
```
Comment (zhengbli) [inline TypeScript.YAML-tmLanguage]: Does this cover the following case (from [ts spec](https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md)):
```
interface KeyValuePair<K, V> extends Array<K | V> { 0: K; 1: V; }
```

Comment (hoanhtien) [inline TypeScript.YAML-tmLanguage]: There is a small problem with type argument i.e. <code>\<K | V\></code> in the example above. This is fixed in the commit below.

**Comment by hoanhtien** [comment]
The new commit fixes all scenarios in https://github.com/Microsoft/TypeScript-Sublime-Plugin/issues/226.

**Comment by zhengbli** [comment]
:+1:

**Pull request merged**